src/models/contributorieslist.h

```diff
@@ -30,22 +69,76 @@ class ContributoriesList : private QMap<QPair<Project*,Models::Rate>*,QList<Cont
      */
     void setAllIdContributories(int idContributory);
 
-    bool getInsert() const;
-    void setInsert(bool getInsert);
+    /**
+     * @brief ContributoriesList::isInsert Return TRUE if an element is
+     * inserting else FALSE
+     * @return boolean
+     */
+    bool isInsert() const;
+
+    /**
+     * @brief ContributoriesList::setInsert Change the state of insertion
+     * @param insert Boolean
+     */
+    void setInsert(bool insert);
 
+    /**
+     * @brief ContributoriesList::getNbProjects Return the number of projects
+     * @return Count number of project
+     */
     int getNbProjects();
+
+    /**
+     * @brief ContributoriesList::getSumRate Return the sum of the
+     * contributories rates
+     * @return Sum of contributories rates
+     */
     double getSumRate();
+
+    /**
+     * @brief ContributoriesList::getSumQuantity Return the sum of quantity
+     * (number of hours) of the Contributories
+     * @return sum of quantity
+     */
     double getSumQuantity();
 
+    /**
+     * @brief ContributoriesList::getCustomer Return the Customers linked to
+     * theses contributories
+     * @return Customer
+     */
     QSharedPointer<Customer> getCustomer();
+
+    /**
+     * @brief ContributoriesList::getProjects List of Projects
+     * @return List of Projects
+     */
     QList<Project *> getProjects();
+
+    /**
+     * @brief ContributoriesList::getAllContributories List of all
+     * contributories (all contributories from all projects)
+     * @return List of all contributories
+     */
     QList<Contributory> *getAllContributories();
+
+    /**
+     * @brief ContributoriesList::getRate
+     * @param project
+     * @return
+     */
     Models::Rate getRate(Models::Project *project);
 
+    /**
+     * @brief ContributoriesList::getDataMap Return a list of Billing and it
+     * value linked which indicates if it is inserting or not
+     * @return List of billing and value linked
+     */
     QVariantList getDataMap();
+
 private:
-    int _idBilling;
-    bool _insert;
+    int _idBilling; //!< Billing ID
+    bool _insert;   //!< an element is inserting
```

Comment:
"is inserted*" sinon en utilisant le présent continu là tu veux dire qu'il est en train de s'insérer ^^